backend/vxlan/vxlan_windows.go

```diff
@@ -191,7 +191,10 @@ func (be *VXLANBackend) RegisterNetwork(ctx context.Context, wg sync.WaitGroup,
 	}
 
 	lease, err = be.subnetMgr.AcquireLease(ctx, subnetAttrs)
```

Comment:
Why are we re-acquiring the lease anyway?

Reply:
From the codes, I can guess, the first time we acquire lease is to get `subnet`. But at that time mac is `nil`, so we need re-acquire the lease when the vxlan network is ready(mac is not nil)

Reply:
Yep that is exactly why. We need to wait until the network is created to pass around the mac